technologies/job/gcp-cloud-functions/project/context.yaml

```diff
@@ -0,0 +1,47 @@
+id: project
+label: project
```

Comment:
project is not the good name here, nothing nothing bound to gcp function UI.

default or functions would be better.